scipy/optimize/_basinhopping.py

```diff
@@ -437,6 +437,19 @@ def basinhopping(func, x0, niter=100, T=1.0, stepsize=0.5,
         `take_step` and `accept_test`, and these functions use random
         number generation, then those functions are responsible for the state
         of their random number generator.
+    accept_rate : float, optional
+        The target acceptance rate that is used to adjust the `stepsize`.
+        If the current acceptance rate is greater than the target,
+        then the `stepsize` is increased. Otherwise, it is decreased.
+        Default is 0.5.
+
+        .. versionadded:: 1.8.0
+
+    factor : float, optional
```

Comment:
What is the sensible value range for factor??

Reply:
> 0 and < 1. 

It might make sense to raise an exception if it is out of that range.